code/modules/mob/living/living.dm

```diff
@@ -562,7 +562,7 @@
 /mob/living/proc/makeTrail(turf/target_turf, turf/start, direction)
 	if(!has_gravity())
 		return
```

Comment:
Why are you keeping this?

Reply:
Whoops. Fixing that now

Reply:
Its quicker to just deactivate most of the code inside the proc rather than going around and working out everywhere that can activate makeTrail. Partially due to the fact that I will be returning to change it all at a later date. Of which will probably keep makeTrail about in a different manner.